kafka-connect-cloud-common/src/main/scala/io/lenses/streamreactor/connect/cloud/common/sink/transformers/Transformers.scala

```diff
@@ -37,22 +38,29 @@ case class SequenceTransformer(transformers: Transformer*) extends Transformer {
 
 }
 
-case class TopicsTransformers(transformers: Map[Topic, Transformer]) extends Transformer {
+case class TopicsTransformers(transformers: Map[Topic, Transformer]) extends Transformer with StrictLogging{
   def get(topic:         Topic): Option[Transformer] = transformers.get(topic)
-  def transform(message: MessageDetail): Either[RuntimeException, MessageDetail] =
-    transformers.get(message.topic).fold(message.asRight[RuntimeException])(_.transform(message))
+  def transform(message: MessageDetail): Either[RuntimeException, MessageDetail] = {
+   logger.info(
+     s"""
+       |***Transformers map: ${transformers.keySet.mkString(",")}
+       |""".stripMargin)
+    transformers.get(message.topic)
+      //check to see if there is a `*`
+      .orElse(transformers.get(Topic("*")))
+      .fold(message.asRight[RuntimeException])(_.transform(message))
+  }
 }
 
-object TopicsTransformers {
+object TopicsTransformers extends StrictLogging {
   def from(bucketOptions: Seq[WithTransformableDataStorage]): TopicsTransformers = {
-
     val transformersMap =
       bucketOptions
-        .filter(_.sourceTopic.nonEmpty)
         .foldLeft(Map.empty[Topic, Transformer]) {
           case (map, bo) =>
             if (bo.dataStorage.hasEnvelope) {
-              val topic = Topic(bo.sourceTopic.get)
+              //replace empty with `*`. This is because CloudSinkBucketOptions moves `*` to empty
+              val topic = Topic(bo.sourceTopic.getOrElse("*"))
```

Comment:
Do you think it would be better if `CloudSinkBucketOptions` was configured differently so that the `*` was more intuitive?  Instead of a generic `Option` a custom type could be introduced.

I'm not suggesting you make any changes but perhaps there is some technical debt here if this tripped you up in the past, and it makes me think that there could be improvements made to make the configuration code more understandable?

Reply:
yes, I think it is probably going to be clear a "default" option if enabled

Reply:
I raised LC-204 on Jira to cover improvements here.